api/src/main/java/net/automatalib/modelchecking/ModelChecker.java

```diff
@@ -80,18 +71,12 @@
      *
      * @see ModelChecker
      */
-    interface MealyModelChecker<I, O, P> extends ModelChecker<I, MealyMachine<?, I, ?, O>, P, Word<O>> {
-
-        @Nullable
-        @Override
-        MealyMachine<?, I, ?, O> findCounterExample(MealyMachine<?, I, ?, O> automaton,
-                                                    Collection<? extends I> inputs, P property)
-                throws ModelCheckingException;
+    interface MealyModelChecker<I, O, P, R> extends ModelChecker<I, MealyMachine<?, I, ?, O>, P, R> {
 
         /**
          * Returns the outputs for which all transitions should be removed.
-         *
-         * That is, before the model checker tries to find a counter example to the automaton every transition
+         * <p>
+         * That is, before the model checker tries to find a counterexample to the automaton every transition
```

Comment:
I am being a bit pedantic probably, but the reason I am writing counter example and not counterexample is because findCounterExample is not written findCounterexample.

Reply:
I think the more wide-spread version is "counterexample", but I can see your argumentation. I can revert this.